app/main/helpers/search_helpers.py

```diff
@@ -191,10 +190,7 @@ def build_search_query(framework, request_args, lot_filters, content_builder, lo
     if 'q' in query:
         query['q'] = replace_g5_search_dots(query['q'])
```

Comment:
We can completely remove this `replace_g5_search_dots` thing now, as it's been a long time since G5 services came back from our index, with or without dashes instead of dots.  Doesn't have to be for this PR, but it's definitely completely pointless these days.

Reply:
Cool, I'll rip it out.